src_firmware/src/storage.c

```diff
@@ -25,11 +26,29 @@ static uint8_t alloc_size;
 
 static char current_template[CONFIG_TEMPLATE_NAME_SIZE];
 
+static bool initialized;
 
-void init_storage(void)
+#pragma region ErrorManagement
+static int errno_error_code;
```

Comment:
```suggestion
static error_codes_t errno_error_code;
```

Reply:
incorrect. This is error from errno file, not from our return_codes.h